data/mainnet/pools.js

```diff
@@ -942,6 +942,166 @@ module.exports = [
     </div>
    `,
   },
+  {
+    chain: CHAINS_ID.ETH_MAINNET,
+    id: 'aura_auraBAL',
+    type: POOL_TYPES.INCENTIVE,
+    tradingApyFunction: {
```

Comment:
I don't think there's any trading APY for this pool. AuraBAL has no underlying growth, it's just earning staking rewards in the form of: AURA, BAL and bb-a-usd. Right?

Reply:
Yes! All the rewards should be calculated in standard apy.